components/pages/donor-entity/ClinicalTimeline/Timeline.tsx

```
@@ -171,6 +171,15 @@ const Timeline = ({
   const theme = useTheme();
   const timelineStyles = React.useMemo(() => getTimelineStyles(theme), [theme]);
 
+  const donorAgeDisplay = (type, data) => {
```

(type: EntityType, data: any) => ...
little sprinkle of TS.. not too sure what data is..

---

thanks for the sprinkles! ;)
yeah this is a tricky one b/c each 'entity' has a different shape for `data`
could be a treatment node, primary diagnosis, specimen, etc., so `data` is basically a union type of many different types
But until I work through all the different Timeline Tab entities and fully update the dummy data, using a union type like that would throw a TS error
so I'll leave `data` blank for now and update that in a future timeline ticket (which could be next sprint anyways)